doc/integrator/make.rst

```diff
@@ -74,14 +74,30 @@ Custom rules
 ------------
 
 In the ``<package>.mk`` file we can create some other rules.
-Here is a simple example that copies a file:
+Here is a simple example that create the `apache/mapcache.xml` file
+and remove it on the `clean` rule:
 
 .. code:: makefile
 
-    DESTINATION_FILE = <destination file>
-    PRE_RULES = $(DESTINATION_FILE)
+    PRE_RULES = apache/mapcache.xml
 
-    $(DESTINATION_FILE): <source file>
-        cp $< $@
+    apache/mapcache.xml: tilegeneration/config.yaml .build/dev-requirements.timestamp
+        $(VENV_BIN)/generate_controller --mapcache
+
+    clean: project-clean
+    .PHONY: project-clean
+    project-clean:
+        rm -tapache/mapcache.xml
+
+The ``tilegeneration/config.yaml`` and ``.build/dev-requirements.timestamp`` files are files that
```

Comment:
I would rather write:

``tilegeneration/config.yaml`` and ``.build/dev-requirements.timestamp`` are the files that must be built to run the current rule.